Update signTask 36552572: Edit signtool docs /fd /td infotool-exe.md (#26325)
dotnet/docs: docs/framework/tools/signtool-exe.md

 This tool is automatically installed with Visual Studio. To run the tool, use [Visual Studio Developer Command Prompt or Visual Studio Developer PowerShell](/visualstudio/ide/reference/command-prompt-powershell).

> [!Note]  

> > A warning (error code 0, initially) will be thrown if /fd is not specified during signing and if /td is not specified during timestamping. In later versions of SignTool, the warning will become an error. " --> An error (error code 1) will be thrown if /fd is not specified during signing and if /td is not specified during timestamping.  

Reviewer: ```suggestion
> The Windows 10 SDK, Windows 10 HLK, Windows 10 WDK and Windows 10 ADK **builds 20236 and later** require specifying the digest algorithm. The SignTool `sign` command requires the `/fd` **file digest algorithm** and the `/td` **timestamp digest algorithm** option to be specified during signing and timestamping, respectively. An error (error code 1) will be thrown if `/fd` is not specified during signing and if `/td` is not specified during timestamping.
```

|`/d`  *Desc*|Specifies a description of the signed content.|  
|`/du`  *URL*|Specifies a Uniform Resource Locator (URL) for the expanded description of the signed content.|  
|`/f`  *SignCertFile*|Specifies the signing certificate in a file. If the file is in Personal Information Exchange (PFX) format and protected by a password, use the `/p` option to specify the password. If the file does not contain private keys, use the `/csp` and `/kc` options to specify the CSP and private key container name.|  
|`/fd`|Specifies the file digest algorithm to use for creating file signatures. </br> **Note:** An error  is generated if the /fd switch is not provided while signing.|

> |`/fd`  *certHash*|Specifying the string *certHash* will default to the algorithm used on the signing certificate. </br> **Note:** An error  is generated if the /fd switch is not provided while signing.|  

Reviewer: ```suggestion
|`/fd`  *certHash*|Specifying the string *certHash* will default to the algorithm used on the signing certificate. </br> **Note:** An error is generated if the `/fd` switch is not provided while signing.|  
```

|`/sha1`  *Hash*|Specifies the SHA1 hash of the signing certificate. The SHA1 hash is commonly specified when multiple certificates satisfy the criteria specified by the remaining switches.|  
|`/sm`|Specifies that a machine store, instead of a user store, is used.|  
|`/t`  *URL*|Specifies the URL of the time stamp server. If this option (or `/tr`) is not present, the signed file will not be time stamped. A warning is generated if time stamping fails. This option cannot be used with the `/tr` option.|  

> |`/td`  *alg*|Used with the `/tr` option to request a digest algorithm used by the RFC 3161 time stamp server. </br> **Note:** An error is generated if /td is not provided while timestamping.|

Reviewer: ```suggestion
|`/td`  *alg*|Used with the `/tr` option to request a digest algorithm used by the RFC 3161 time stamp server. </br> **Note:** An error is generated if `/td` is not provided while timestamping.|
```

> |`/td`  *alg*|Used with the `/tr` option to request a digest algorithm used by the RFC 3161 time stamp server. </br> **Note:** An error is generated if /td is not provided while timestamping." (there is no default anymore <- does not need to go in doc. |

Reviewer: "the does not need to go in doc" was not supposed to go in the doc

```suggestion
|`/td`  *alg*|Used with the `/tr` option to request a digest algorithm used by the RFC 3161 time stamp server. </br> **Note:** An error is generated if `/td` is not provided while timestamping.|
```

|----------------------|-----------------|  
|`/p7`|Time stamps PKCS #7 files.|  
|`/t`  *URL*|Specifies the URL of the time stamp server. The file being time stamped must have previously been signed. Either the `/t` or the `/tr` option is required.|  

> |`/td`  *alg*|Used with the `/tr` option to request a digest algorithm used by the RFC 3161 time stamp server. </br> **Note:** An error is generated if /td is not provided while timestamping.|

Reviewer: ```suggestion
|`/td`  *alg*|Used with the `/tr` option to request a digest algorithm used by the RFC 3161 time stamp server. </br> **Note:** An error is generated if `/td` is not provided while timestamping.|
```

> |`/td`  *alg*|Used with the `/tr` option to request a digest algorithm used by the RFC 3161 time stamp server. </br> **Note:** An error is generated if /td is not provided while timestamping." (there is no default anymore <- does not need to go in doc. |

Reviewer: See above requested changes please
